javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,308 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  // Solution code here...
+  let plusTwo=[];
+  for(let i=0; i<arr.length; i++){
+    plusTwo.push((num) => num + 2);
+  } console.log(plusTwo);
+  return plusTwo;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named typeNum that, given an array as input, uses filter to return an array containing only the numbers.
+
+For example, typeNum([1, 'bob' ,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const typeNum = (arr) => {
+  // Solution code here...
+  let onlyNum= arr.filter(
+    element => typeof element === 'number'
+  );
+  console.log(onlyNum);
+}
+return (onlyNum);
```

Comment:
This one would be passing if you move the return up, inside of the curly braces and add a semi-colon to the end of the function. 

```suggestion
const typeNum = (arr) => {
  let onlyNum= arr.filter(
    element => typeof element === 'number'
  );
  console.log(onlyNum);
return (onlyNum);
};
```